### ISSUE TITLE
Allow for default cloudfront TTLs

### ISSUE THREAD

**Issue by icco** [terraform-providers/terraform-provider-aws#1472]
### Terraform Version

Terraform v0.10.2

### Affected Resource(s)

 - cloudfront_distrobution

### Terraform Configuration Files
```hcl
resource "aws_cloudfront_distribution" "test" {
  origin {
    domain_name = "https://google.com"
    origin_id   = "google"
  }

  enabled         = true
  is_ipv6_enabled = true

  aliases = ["mysite.example.com", "yoursite.example.com"]

  default_cache_behavior {
    allowed_methods  = ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"]
    cached_methods   = ["GET", "HEAD"]
    target_origin_id = "google"

    forwarded_values {
      query_string = false

      cookies {
        forward = "none"
      }
    }

    viewer_protocol_policy = "allow-all"
  }

  viewer_certificate {
    cloudfront_default_certificate = true
  }

  restrictions {
    geo_restriction {
      restriction_type = "none"
    }
  }
}
```

### Expected Behavior

`terraform validate` returns nothing.

### Actual Behavior

```
$ terraform validate
There are warnings and/or errors related to your configuration. Please
fix these before continuing.

Errors:

  * aws_cloudfront_distribution.test: "default_cache_behavior.0.default_ttl": required field is not set
  * aws_cloudfront_distribution.test: "default_cache_behavior.0.max_ttl": required field is not set
  * aws_cloudfront_distribution.test: "default_cache_behavior.0.min_ttl": required field is not set
```

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:

1. `terraform validate`

### Important Factoids

This is more of a feature request, but cloudfront seems to [not require these values anymore](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/Expiration.html).

**Comment by bflad** [comment]
For cross-reference this is the same request as #1994

**Comment by bflad** [comment]
A pull request to make these attributes optional can be found here: #3571

**Comment by bflad** [comment]
Support to make all 3 TTL arguments optional has been merged into master and will be released in v1.11.0 of the AWS provider, likely at the end of the week. Please note they are default_ttl (1 day), max_ttl (365 days), and min_ttl (0 days) to match the behavior of the API and console.

**Comment by bflad** [comment]
This has been released in [version 1.11.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1110-march-09-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
